README.md

```diff
@@ -3,56 +3,145 @@
 ## Objectives 
 + Explain how to make an API request using AJAX
 + Make an API `get` request using AJAX
-+ Make an API `post` request using AJAX
++ Make an API post request using AJAX
 + Parse JSON
 + Explain what JSON is
 
-
 ## Intro
 
-Now that we've made `get` requests to a website for text, let's do something more interesting with AJAX, let's use it to access an API.
-
+Now that we've made `get` requests to our server for text files, let's do something more interesting with AJAX by using it to access an API. Before we do that, let's talk about JSON.
```

Comment:
Have intro talk more about the pain of dealing with data that isn't formatted nicely in JSON. want to prove the pain to students before we provide the easy-peasy solution that JSON provides. Something like it's hard to parse through a long string of data, etc.